test/unit/lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -310,10 +373,6 @@ describe('EventBridgeEvents', () => {
       let eventBusLogicalId;
       let ruleResource;
       let ruleTarget;
-      let inputPathRuleTarget;
-      let inputTransformerRuleTarget;
-      let disabledRuleResource;
-      let enabledRuleResource;
```

Comment:
I couldn't resist moving these `let` variables out of here into the tests that need them.